catalog/spatial/registry/registry-common/src/test/java/org/codice/ddf/registry/common/metacard/TestRegistryObjectMetacardType.java

```diff
@@ -61,7 +48,7 @@ public void testConstruction() {
         assertThat(descriptors, not(nullValue()));
         assertThat(CollectionUtils.isEmpty(descriptors), is(false));
 
-        assertThat(registryObjectMetacardType.getAttributeDescriptor(Metacard.ID)
+        assertThat(registryObjectMetacardType.getAttributeDescriptor(RegistryObjectMetacardType.REGISTRY_ID)
```

Comment:
My comment was in reference to accessing that attribute in this test class specifically. We do have access to the standard MetacardType attributes when creating an actual metacard as @brendan-hofmann referenced.